src/SFA.DAS.Assessor.Functions/Domain/Print/DeliveryNotificationCommand.cs

```diff
@@ -69,6 +71,19 @@ private async Task ProcessFile(string fileName)
                 return;
             }
 
+            var invalidDeliveryNotificationStatuses = receipt.DeliveryNotifications
+                   .GroupBy(certificateDeliveryNotificationStatus => certificateDeliveryNotificationStatus.Status)
+                   .Select(certificateDeliveryNotificationStatus => certificateDeliveryNotificationStatus.Key)
+                   .Where(deliveryNotificationStatus => !DeliveryNotificationStatus.Contains(deliveryNotificationStatus))
+                   .ToList();
+
+            //Scenario 4:  Invalid status in delivery notification file
```

Comment:
Comments like this don't actually refer to anything useful to reword them or remove them

Reply:
removed